public/theme/skeleton/partials/_master.twig

```diff
@@ -7,9 +7,9 @@
         {# make sure we always display a proper title: The record's title if there is one, appended with the
            sitename. If there is no title, we append the sitename with the payoff, if there is one. #}
         <title>
-            {%- if record.hasField('title') %}{{ record.title|striptags }} | {% endif -%}
+            {%- if record.hasField('title')|default() %}{{ record.title|striptags }} | {% endif -%}
```

Comment:
why default?

Reply:
Because I wrote this before #351 was merged. :-) 
I'll fix.